pallets/marketplace/src/tests/extrinsics.rs

```diff
@@ -599,3 +691,195 @@ fn set_commission_fee_unhappy() {
+    ExtBuilder::default()
+        .caps(vec![(ALICE, 10000)])
+        .build()
+        .execute_with(|| {
+            let alice: mock::Origin = RawOrigin::Signed(ALICE).into();
+            assert_eq!(Marketplace::marketplace_id_generator(), 0);
+            assert_eq!(Marketplace::marketplaces(1), None);
+
+            let fee = 25;
+            let name = vec![50];
+            let kind = MPT::Public;
+            let uri = Some(vec![66]);
+            let raw_too_short_uri: URI = vec![];
+            let raw_too_long_uri: URI = [0; 1001].to_vec();
+            let info = MarketplaceInformation::new(
+                kind,
+                fee,
+                ALICE,
+                vec![],
+                name.clone(),
+                uri.clone(),
+                uri.clone(),
+            );
+
+            assert_ok!(Marketplace::create(
+                alice.clone(),
+                kind.clone(),
+                fee,
+                name.clone(),
+                uri.clone(),
+                uri.clone()
+            ));
+            assert_eq!(Marketplace::marketplace_id_generator(), 1);
+            assert_eq!(Marketplace::marketplaces(1), Some(info));
+            let nok = Marketplace::update_uri(alice.clone(), 1, raw_too_short_uri);
+            assert_noop!(nok, Error::<Test>::TooShortMarketplaceUri);
+            let nok = Marketplace::update_uri(alice, 1, raw_too_long_uri);
+            assert_noop!(nok, Error::<Test>::TooLongMarketplaceUri);
+        })
+}
+
+#[test]
+fn update_logo_uri_happy() {
+    ExtBuilder::default()
+        .caps(vec![(ALICE, 10000)])
+        .build()
+        .execute_with(|| {
+            let alice: mock::Origin = RawOrigin::Signed(ALICE).into();
+            assert_eq!(Marketplace::marketplace_id_generator(), 0);
+            assert_eq!(Marketplace::marketplaces(1), None);
+
+            let fee = 25;
+            let name = vec![50];
+            let kind = MPT::Public;
+            let uri = Some(vec![66]);
+            let updated_uri = Some(vec![67]);
+            let info = MarketplaceInformation::new(
+                kind,
+                fee,
+                ALICE,
+                vec![],
+                name.clone(),
+                uri.clone(),
+                uri.clone(),
+            );
+            let updated_info = MarketplaceInformation::new(
+                kind,
+                fee,
+                ALICE,
+                vec![],
+                name.clone(),
+                uri.clone(),
+                updated_uri.clone(),
+            );
+
+            assert_ok!(Marketplace::create(
+                alice.clone(),
+                kind.clone(),
+                fee,
+                name.clone(),
+                uri.clone(),
+                uri.clone()
+            ));
+            assert_eq!(Marketplace::marketplace_id_generator(), 1);
+            assert_eq!(Marketplace::marketplaces(1), Some(info));
+            assert_ok!(Marketplace::update_logo_uri(
```

Comment:
Before this add a one liner that checks that the current logo uri isn't the same as the future one.
Something like this:
assert_ne!(Marketplace::marketplaces(1).unwrap().logo_uri, future_logo_uri)